lib/core/src/Cardano/Wallet/Primitive/Types/Tx.hs

```diff
@@ -398,12 +400,27 @@ instance Buildable a => Buildable (WithDirection a) where
         <> (case d of; Incoming -> "+"; Outgoing -> "-")
         <> build a
 
--- | @SealedTx@ is a serialised transaction that is ready to be submitted
--- to the node.
+-- | @SealedTx@ is a signed and serialised transaction that is ready to be
+-- submitted to the node.
 newtype SealedTx = SealedTx { getSealedTx :: ByteString }
     deriving stock (Show, Eq, Generic)
     deriving newtype (ByteArrayAccess, NFData)
 
+-- | A serialised transaction that may be only partially signed, or even
+-- invalid.
+newtype SerialisedTx = SerialisedTx { payload :: ByteString }
+    deriving stock (Show, Eq, Generic, Ord)
+    deriving newtype (Semigroup, Monoid, ByteArray, ByteArrayAccess, NFData)
+
+-- | @SerialisedTxParts@ is a serialised transaction body, and a possibly
+-- incomplete set of serialised witnesses, along with an encoding of the
+-- combined transaction.
+data SerialisedTxParts = SerialisedTxParts
```

Comment:
Why is `SerialisedTxParts` needed when we have `SerialisedTx`?

Isn't the structure it exposes already unambiguously exposed in the raw CBOR?

Reply:
Yes, but I think it will be helpful for API consumers to get both.